### PR TITLE
Create dominic.cpp - 2405: Optimal Partition of String

### PR DESCRIPTION
**Pull request by DominicS99**

### CODE REVIEW

File: Week 4/Extra Practice Problems (Week 4)/2405: Optimal Partition of String/dominic.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Fun and faster (but a bit more complicated) bitwise
+class Solution {
+public:
+    int partitionString(string s) {
+        int n = 0;
+        int res = 1;
+        for (char &c : s) {
+            int curr = (2 << (c-'a'));
```
Comment (donaldong) [inline Week 4/Extra Practice Problems (Week 4)/2405: Optimal Partition of String/dominic.cpp:8]: why not shift 1?

**Review by donaldong — APPROVED** [review]
LOL, very nice!

File: Week 4/Extra Practice Problems (Week 4)/2405: Optimal Partition of String/dominic.cpp (hunk 1 above)
Comment (DominicS99) [reply Week 4/Extra Practice Problems (Week 4)/2405: Optimal Partition of String/dominic.cpp:8]: Oh, I meant to put 1 there. Nice catch
```suggestion
            int curr = (1 << (c-'a'));
```

**Review by DominicS99 — COMMENTED** [review]
Changed to a shift 1 rather than shift 2

**Pull request merged**